sample/build.gradle

```
@@ -96,10 +102,12 @@ keeper {
   // Uncomment this line to debug the R8 from a remote instance.
   //r8JvmArgs.addAll(Arrays.asList("-Xdebug", "-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=y"))
 
-  // Example: enable the new experimental R8 entry-point.
-  //traceReferences {
-    //arguments.set(Arrays.asList("--map-diagnostics:MissingDefinitionsDiagnostic", "error", "info"))
-  //}
+  if (useTraceReferences) {
+    // Enable the new experimental TraceReferences tool.
+    traceReferences {
+      arguments = ["--map-diagnostics:MissingDefinitionsDiagnostic", "error", "info"]
```

From reading https://issuetracker.google.com/issues/173435379#comment4, it sounds like this option basically makes missing definition nonfatal -- might be worth mentioning that?

```suggestion
      // Don't fail the build if missing definitions are found.
      arguments = ["--map-diagnostics:MissingDefinitionsDiagnostic", "error", "info"]
```